python/prima/_bounds.py

```diff
@@ -20,7 +8,6 @@ def process_bounds(bounds, lenx0):
     fewer entries than the length of x0, the remaining entries will generated as -/+ infinity.
     Some examples of valid lists of tuple, assuming len(x0) == 3:
     [(0, 1), (2, 3), (4, 5)] -> returns [0, 2, 4], [1, 3, 5]
-    [(0, 1), (2, 3), None]   -> returns [0, 2, -inf], [1, 3, inf]
```

Comment:
Why is `[(0, 1), (2, 3), None]   -> returns [0, 2, -inf], [1, 3, inf]`  not supported anymore?

Reply:
It turns out if wasn't supported in the first place. I looked to see if SciPy supports it, and they don't, so I decided that instead of implementing it I would follow SciPy.